README.md

```diff
@@ -84,7 +87,7 @@
 ## 🚀 Live Demo <a name="live-demo"></a>
 
 > - [Live Demo on Gh-pages](https://ab-noori.github.io/math_magicians/)
-> - [Live Demo on Render](https://math-magicians-ab.onrender.com)
+> - [Live Demo on Render](https://math-magicians-fans.onrender.com)
```

Comment:
- [ ] [OPTIONAL] You have an awesome UI @ab-noori  🥇  However, kindly note that you have done a great job deploying this application to GitHub pages and on-render, however, you deleted the working URL for the on-render deployment and replaced it with the URL that is not working. Please replace with: https://math-magicians-ab.onrender.com

Reply:
Hi @Whoistolu  👋,
Thank you so much 🙏  for reviewing and approving my project.
Your nice and constructive feedback is appreciated 🙏  and I will consider that.